build/ci/.azure-devops-package-net6-win.yml

```diff
@@ -65,15 +65,13 @@ jobs:
 
   - template: templates/gitversion.yml
 
-  - template: templates/dotnet6-mobile-install-windows.yml
-  - template: templates/dotnet7-mobile-install-windows.yml
+  - ${{ if and(succeeded(), ne(variables.UnoDisableNet6Mobile, 'true')) }}:
+    - template: templates/dotnet6-mobile-install-windows.yml
+  - ${{ if and(succeeded(), ne(variables.UnoDisableNet7Mobile, 'true')) }}:
+    - template: templates/dotnet7-mobile-install-windows.yml
```

Comment:
![image](https://user-images.githubusercontent.com/31348972/202294363-646debef-97ac-4466-ba49-355531364899.png)

Will need to know the correct syntax when I get back to this PR.